### ISSUE TITLE
Cannot install gpg key on Ubuntu 22.04

### ISSUE THREAD

**Issue by dany-nonstop** [christgau/wsdd#138]
copied this apt-key command from `README.md` but see that the key is not fetched. 

```bash
$ sudo apt-key adv --fetch-keys https://pkg.ltec.ch/public/conf/ltec-ag.gpg.key
Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).
Executing: /tmp/apt-key-gpghome.sdHKSoGeao/gpg.1.sh --fetch-keys https://pkg.ltec.ch/public/conf/ltec-ag.gpg.key
gpg: requesting key from 'https://pkg.ltec.ch/public/conf/ltec-ag.gpg.key'
gpg: WARNING: unable to fetch URI https://pkg.ltec.ch/public/conf/ltec-ag.gpg.key: No such file or directory
```
looked up and figured out the right way to install a gpg key is actually to first download it, then have it dearmored, installed into the right directory.

```bash
$ wget https://pkg.ltec.ch/public/conf/ltec-ag.gpg.key
$ gpg --dearmor ltec-ag.gpg.key
$ sudo install -o root -g root -m 644 ltec-ag.gpg.key.gpg /etc/apt/trusted.gpg.d/
```

hope this would be helpful for people with my problems, especially for future installations when apt-key is totally abandoned.

**Comment by christgau** [comment]
Thanks for the clarification, but there is actually no need to install the third-party repository. wsdd has landed in the Universe repository (see #137).

**Comment by dany-nonstop** [comment]
Hi, @christgau I was not aware of it! 👍 I just removed my source list and gpg key and reinstalled from Ubuntu official. Many thanks for the great work!
